Does this require an update to heartbeats? I tried it locally (and put an updated cmake in the s3 bucket), but got:
```
error: failed to run custom build command for `heartbeats-simple-sys v0.4.1`
process didn't exit successfully: `C:\Users\lars\code\servo\target\debug\build\heartbeats-simple-sys-bf7003c17df5d8c7\build-script-build` (exit code: 101)
--- stdout
running: "cmake" "C:\\Users\\lars\\code\\servo\\.cargo\\registry\\src\\github.com-1ecc6299db9ec823\\heartbeats-simple-sys-0.4.1/heartbeats-simple" "-G" "Visual Studio 15 Win64" "-DCMAKE_INSTALL_PREFIX=C:\\Users\\lars\\code\\servo\\target\\debug\\build\\heartbeats-simple-sys-28376003c39ead8c\\out" "-DCMAKE_C_FLAGS= /nologo /MD" "-DCMAKE_C_FLAGS_RELEASE= /nologo /MD" "-DCMAKE_CXX_FLAGS= /nologo /MD" "-DCMAKE_CXX_FLAGS_RELEASE= /nologo /MD" "-DCMAKE_BUILD_TYPE=Release"

--- stderr
CMake Error: Could not create named generator Visual Studio 15 Win64

Generators
  Visual Studio 14 2015 [arch] = Generates Visual Studio 2015 project files.
                                 Optional [arch] can be "Win64" or "ARM".
  Visual Studio 12 2013 [arch] = Generates Visual Studio 2013 project files.
                                 Optional [arch] can be "Win64" or "ARM".
  Visual Studio 11 2012 [arch] = Generates Visual Studio 2012 project files.
                                 Optional [arch] can be "Win64" or "ARM".
  Visual Studio 10 2010 [arch] = Generates Visual Studio 2010 project files.
                                 Optional [arch] can be "Win64" or "IA64".
  Visual Studio 9 2008 [arch]  = Generates Visual Studio 2008 project files.
                                 Optional [arch] can be "Win64" or "IA64".
  Visual Studio 8 2005 [arch]  = Generates Visual Studio 2005 project files.
                                 Optional [arch] can be "Win64".
```

--- a/mach.bat
+++ b/mach.bat
@@ -1,6 +1,22 @@
 @echo off
 
-SET VS_VCVARS=%VS140COMNTOOLS%..\..\VC\vcvarsall.bat
+IF EXIST "%ProgramFiles(x86)%" (
+  set "ProgramFiles32=%ProgramFiles(x86)%"
+) ELSE (
+  set "ProgramFiles32=%ProgramFiles%"
+)
+
+set VC14VARS=%VS140COMNTOOLS%..\..\VC\vcvarsall.bat
+IF EXIST "%VC14VARS%" (
+  set "VS_VCVARS=%VC14VARS%"
+) ELSE (
+  for %%e in (Enterprise Professional Community) do (
+    IF EXIST "%ProgramFiles32%\Microsoft Visual Studio\2017\%%e\VC\Auxiliary\Build\vcvarsall.bat" (
+      set "VS_VCVARS=%ProgramFiles32%\Microsoft Visual Studio\2017\%%e\VC\Auxiliary\Build\vcvarsall.bat"
+    )
+  )
+)
+
 IF EXIST "%VS_VCVARS%" (
   IF NOT DEFINED Platform (
     IF EXIST "%ProgramFiles(x86)%" (
@@ -11,8 +27,8 @@ IF EXIST "%VS_VCVARS%" (
     )
   )
 ) ELSE (
-  ECHO Visual Studio 2015 is not installed.
-  ECHO Download and install Visual Studio 2015 from https://www.visualstudio.com/
+  ECHO Visual Studio 2015 or 2017 is not installed.
+  ECHO Download and install Visual Studio 2015 or 2017 from https://www.visualstudio.com/
   EXIT /B
 )
 

--- a/python/servo/bootstrap.py
+++ b/python/servo/bootstrap.py
@@ -5,6 +5,7 @@
 from __future__ import absolute_import, print_function
 
 from distutils.spawn import find_executable
+from distutils.version import StrictVersion
 import json
 import os
 import platform
@@ -204,10 +205,19 @@ def windows_msvc(context, force=False):
     def package_dir(package):
         return os.path.join(deps_dir, package, version(package))
 
+    def check_cmake(version):
+        cmake_path = find_executable("cmake")
+        if cmake_path:
+            cmake = subprocess.Popen([cmake_path, "--version"], stdout=PIPE)
+            cmake_version = cmake.stdout.read().splitlines()[0].replace("cmake version ", "")
+            if StrictVersion(cmake_version) >= StrictVersion(version):
+                return True
+        return False
+
     to_install = {}
     for package in packages.WINDOWS_MSVC:
         # Don't install CMake if it already exists in PATH
-        if package == "cmake" and find_executable(package):
+        if package == "cmake" and check_cmake(version("cmake")):
             continue
 
         if not os.path.isdir(package_dir(package)):

--- a/python/servo/packages.py
+++ b/python/servo/packages.py
@@ -18,7 +18,7 @@ WINDOWS_GNU = set([
 ])
 
 WINDOWS_MSVC = {
-    "cmake": "3.6.1",
+    "cmake": "3.7.2",
     "moztools": "0.0.1-5",
     "ninja": "1.7.1",
     "openssl": "1.1.0e-vs2015",

--- a/python/servo/util.py
+++ b/python/servo/util.py
@@ -50,7 +50,7 @@ def host_triple():
     cpu_type = platform.machine().lower()
     if os_type.endswith("-msvc"):
         # vcvars*.bat should set it properly
-        platform_env = os.environ.get("PLATFORM")
+        platform_env = os.environ.get("PLATFORM").upper()
         if platform_env == "X86":
             cpu_type = "i686"
         elif platform_env == "X64":

